pkg/scheduler/algorithm/predicates/error.go

```diff
@@ -30,31 +30,31 @@ var (
-	ErrPodAffinityRulesNotMatch              = newPredicateFailureError("PodAffinityRulesNotMatch")
-	ErrPodAntiAffinityRulesNotMatch          = newPredicateFailureError("PodAntiAffinityRulesNotMatch")
-	ErrExistingPodsAntiAffinityRulesNotMatch = newPredicateFailureError("ExistingPodsAntiAffinityRulesNotMatch")
-	ErrTaintsTolerationsNotMatch             = newPredicateFailureError("PodToleratesNodeTaints")
-	ErrPodNotMatchHostName                   = newPredicateFailureError("HostName")
-	ErrPodNotFitsHostPorts                   = newPredicateFailureError("PodFitsHostPorts")
-	ErrNodeLabelPresenceViolated             = newPredicateFailureError("CheckNodeLabelPresence")
-	ErrServiceAffinityViolated               = newPredicateFailureError("CheckServiceAffinity")
-	ErrMaxVolumeCountExceeded                = newPredicateFailureError("MaxVolumeCount")
-	ErrNodeUnderMemoryPressure               = newPredicateFailureError("NodeUnderMemoryPressure")
-	ErrNodeUnderDiskPressure                 = newPredicateFailureError("NodeUnderDiskPressure")
-	ErrNodeOutOfDisk                         = newPredicateFailureError("NodeOutOfDisk")
-	ErrNodeNotReady                          = newPredicateFailureError("NodeNotReady")
-	ErrNodeNetworkUnavailable                = newPredicateFailureError("NodeNetworkUnavailable")
-	ErrNodeUnschedulable                     = newPredicateFailureError("NodeUnschedulable")
-	ErrNodeUnknownCondition                  = newPredicateFailureError("NodeUnknownCondition")
-	ErrVolumeNodeConflict                    = newPredicateFailureError("VolumeNodeAffinityConflict")
-	ErrVolumeBindConflict                    = newPredicateFailureError("VolumeBindingNoMatch")
+	ErrDiskConflict                          = newPredicateFailureError("NoDiskConflict", "node(s) had no available disk")
+	ErrVolumeZoneConflict                    = newPredicateFailureError("NoVolumeZoneConflict", "node(s) had no available volume zone")
+	ErrNodeSelectorNotMatch                  = newPredicateFailureError("MatchNodeSelector", "node(s) didn't match node selector")
+	ErrPodAffinityNotMatch                   = newPredicateFailureError("MatchInterPodAffinity", "node(s) didn't match pod affinity/anti-affinity")
+	ErrPodAffinityRulesNotMatch              = newPredicateFailureError("PodAffinityRulesNotMatch", "node(s) didn't match pod affinity rules")
+	ErrPodAntiAffinityRulesNotMatch          = newPredicateFailureError("PodAntiAffinityRulesNotMatch", "node(s) didn't match pod anti-affinity rules")
+	ErrExistingPodsAntiAffinityRulesNotMatch = newPredicateFailureError("ExistingPodsAntiAffinityRulesNotMatch", "node(s) didn't satisfy existing pods anti-affinity rules")
+	ErrTaintsTolerationsNotMatch             = newPredicateFailureError("PodToleratesNodeTaints", "node(s) had taints that the pod didn't tolerate")
+	ErrPodNotMatchHostName                   = newPredicateFailureError("HostName", "no available nodes match the requested hostname")
```

Comment:
In the final output of scheduler, all these messages show up with the number of failed nodes in front of them. This would look like "100 no available nodes match the requested hostname". Please chagne to "node(s) didn't match the requested hostname".